cmd/deplist/deplist.go

```diff
@@ -5,20 +5,26 @@ import (
 	"fmt"
 
 	"github.com/RedHatProductSecurity/deplist"
+	"github.com/RedHatProductSecurity/deplist/internal/scan"
 	purl "github.com/mcoops/packageurl-go"
 	log "github.com/sirupsen/logrus"
 )
 
 func main() {
 	deptypePtr := flag.Int("deptype", -1, "golang, nodejs, python etc")
 	debugPtr := flag.Bool("debug", false, "debug logging (default false)")
+	rubyDebug := flag.Bool("rbdebug", false, "ruby version output and then quit")
```

Comment:
Could we use the existing `-debug` option instead of adding a new one? Perhaps in combination with using `exec.LookPath()` in `GetInstalledRubyVersions()` to skip over the rbenv step so that the function is a bit cheaper.

Nitpick: the help text says it quits after printing ruby versions, but I don't think it does